src/Tribe/Admin_List.php

```diff
@@ -181,16 +181,22 @@ public static function filter_by_aggregator_record( array $clauses, WP_Query $wp
 			}
 
 			global $wpdb;
-			
+
```

Comment:
Looks like indentation is a bit off over here.

Reply:
Fixed